### ISSUE TITLE
Generating a model shows no positive output, just warnings

### ISSUE THREAD

**Issue by aaronmarkham** [awslabs/mxnet-model-server#444]
The model is created but by the output I think something is wrong. This is not a nice user experience. 

```
(keras-mxnet) ubuntu@ip-172-31-78-166:~/SmileCNN/keras-mms$ mxnet-model-export --model-name smileCNN --model-path . --service-file-path custom_service.py
/home/ubuntu/anaconda3/envs/keras-mxnet/lib/python3.7/site-packages/h5py/_hl/base.py:19: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import (Mapping, MutableMapping, KeysView,
/home/ubuntu/anaconda3/envs/keras-mxnet/lib/python3.7/site-packages/sklearn/utils/__init__.py:4: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Sequence
```

**Comment by vdantu** [comment]
@aaronmarkham : I would like to understand the error a little better. I think this is happening because of the validation of the custom service file. If that hypothesis is correct, these warnings will appear even when launching this model with the MMS. Did you see this even when hosting with MMS? 
In my opinion, we shouldn't suppress these warnings and we should continue showing them as they can be useful to customers. What would your suggestion be?

**Comment by frankfliu** [comment]
This has been addressed in MMS 1.0
